includes/elements/container.php

```diff
@@ -317,7 +306,6 @@ protected function register_container_layout_controls() {
 					'{{WRAPPER}}' => '{{VALUE}}',
 				],
 				'selectors_dictionary' => [
-					'boxed' => '--width: 100%;',
 					'full' => '--content-width: 100%;',
```

Comment:
I have removed the default value 'boxed width 100%', because that value is set inside scss already.

Reply:
Another thing - currently, this control either sets `--width: 100%` or `--content-width: 100%`.
Now that `--width: 100%` is set by default in the CSS, choosing `full` will set the `--content-width` variable, but it won't modify the `--width` variable. 

The default CSS for containers is:
```
--content-width: Min( 100%, var( --container-max-width, 1140px ) );
--width: 100%;
```

So depending on the value of `--container-max-width`, there could be a clash here
Maybe it would be better that the `boxed` option would set the `--width` variable and also set `--content-width` to `inherit` or `initial` or whatever gives it its original value, and the opposite for the `full` option.

Reply:
@udidol 

About your comment: 

> 
Maybe it would be better that the boxed option would set the --width variable and also set --content-width to inherit or initial or whatever gives it its original value, and the opposite for the full option.

For the `boxed` version we need this defition: `--content-width: Min( 100%, var( --container-max-width, 1140px ) );`
We shouldn't overwrite this.

Reply:
@udidol 

For clarity: the only thing which I have removed is the `--width: 100%` for `boxed` generated by the widget controls. 
This won't change anything, because there is a default `--width` inside the html already.